cms/templates/admin/base_site.html

```diff
@@ -15,5 +15,5 @@ <h1 id="site-name"><a href="{% url 'admin:index' %}">{{ site_header|default:_('D
             <a href="{{ docsroot }}">{% trans 'Documentation' as tmsg %} {{tmsg|force_escape}}</a> /
         {% endif %}
     {% endif %}
-    <a href="{% url 'admin:logout' %}">{% trans 'Log out' as tmsg %} {{tmsg|force_escape}}</a>
+    <a href="{% url 'logout' %}/">{% trans 'Log out' as tmsg %} {{tmsg|force_escape}}</a>
```

Comment:
it gives `/logout` and loads logout.html page here it needs trailing slash.

Reply:
Yeah, that's my concern though -- the `/logout/` and `/logout` paths come from different places. I think I'd rather hardcode than add a slash to the result of a reverse lookup, just because that slash would be so easy to miss. Something we need to clean up later.